app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterFragment.kt

```diff
@@ -28,20 +33,36 @@ class PhoneNumberEnterFragment :
     companion object {
         const val REQUEST_KEY = "REQUEST_KEY"
         const val RESULT_KEY = "RESULT_KEY"
-        fun create(): PhoneNumberEnterFragment = PhoneNumberEnterFragment()
+
+        fun create(countryCode: CountryCode? = null, phoneNumber: String? = null): PhoneNumberEnterFragment =
+            PhoneNumberEnterFragment().withArgs(
+                ARG_COUNTRY_CODE to countryCode,
+                ARG_PHONE_NUMBER to phoneNumber
+            )
     }
 
     override val presenter: PhoneNumberEnterContract.Presenter by inject()
     private val binding: FragmentPhoneNumberEnterBinding by viewBinding()
+    private val phoneNumber by args<String?>(ARG_PHONE_NUMBER, null)
+    private var countryCode by args<CountryCode?>(ARG_COUNTRY_CODE, null)
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) = with(binding) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        binding.initViews()
```

Comment:
initViews -> initView. 
the parent method is named "onViewCreated", so we init root view

Reply:
🤔🤔🤔 I always thought about initing views, like textView, imageView, etc that are inside current fragment